src/interface.jl

```diff
@@ -126,6 +135,21 @@ _alloc_ed2(basis::AbstractPoly4MLBasis, x) =
       _alloc(basis, x), _alloc_d(basis, x), _alloc_dd(basis, x)
 
 
+# special functions for SparseProduct
```

Comment:
Is there a reason to keep this block of comments or could it be removed?

Reply:
I think this can be removed.